factors/dependencies.md

```diff
@@ -22,10 +22,12 @@ Apple provides an excellent guide on the [Swift Package Manager](https://swift.o
 
 You can use a [.xcode-version](https://github.com/fastlane/ci/blob/master/docs/xcode-version.md) file in the root of your iOS project to declare the exact version of Xcode to be used for a given iOS app.
 
-This way, you can configure your CI-system to automatically install and use a given Xcode version. 
+This way, you can configure your CI-system to automatically install and use a given Xcode version. You can also install [chxcode] to automatically switch Xcode versions.
```

Comment:
Maybe use the inline links like we do in the rest of the markdown files. Also, maybe mention that it only switches Xcode, and doesn't install it.

> To just switch the Xcode version (assuming you already have it installed), you can use a tool like [chxcode](https://github.com/klaaspieter/chxcode)